pom.xml

```diff
@@ -224,6 +224,11 @@
                 <artifactId>nimbus-jose-jwt</artifactId>
                 <version>${nimbusds.version}</version>
             </dependency>
+            <dependency>
+                <groupId>com.nimbusds</groupId>
+                <artifactId>oauth2-oidc-sdk</artifactId>
+                <version>8.28.3</version>
```

Comment:
move version separated and use a placeholder

Reply:
fixed in 789a83b